db.json

```diff
@@ -1,5 +1,35 @@
 {
   "workstreams": [
+    {
+      "id": "browser-testing-api",
+      "name": "Browser Testing API",
+      "scope": "Internal APIs for automating testing of web platform features implemented in web browsers, e.g., to trigger JavaScript garbage collection.",
+      "editors": [
+        {
+          "name": "James Graham",
+          "email": "jgraham@mozilla.com"
```

Comment:
If it's not important to use a corp-affiliated email, I'd prefer to use `james@hoppipolla.co.uk` (also below).

Reply:
Will do:

```suggestion
          "email": "james@hoppipolla.co.uk"
```